testdata/baselines/reference/submodule/compiler/blockScopedEnumVariablesUseBeforeDef_verbatimModuleSyntax.js.diff

```diff
@@ -0,0 +1,18 @@
+--- old.blockScopedEnumVariablesUseBeforeDef_verbatimModuleSyntax.js
++++ new.blockScopedEnumVariablesUseBeforeDef_verbatimModuleSyntax.js
+@@= skipped -29, +29 lines =@@
+ }
+ function foo2() {
+     return E.A;
+-    let E;
```

Comment:
This looks like a regression to investigate

Reply:
I think this is related to `ShouldPreserveConstEnums`, which doesn’t account for `verbatimModuleSyntax`, while that case is handled in

https://github.com/microsoft/TypeScript/blob/3eb7b6a1794a6d2cde7948a3016c57e628b104b9/src/compiler/utilities.ts#L9124-L9129

https://github.com/microsoft/TypeScript/blob/3eb7b6a1794a6d2cde7948a3016c57e628b104b9/src/compiler/utilities.ts#L9029-L9034